cclib/io/ccio.py

```diff
@@ -62,6 +62,11 @@
 except ImportError:
     _has_cclib2openbabel = False
 
+try:
+    import pandas as pd
+except ImportError:
+    # Fail silently for now.
+    pass
```

Comment:
Is there an alternative to failing silently? Maybe setting a variable that can be used in `ccframe` to return cleanly instead of failing when trying to access non-existent `pd`?

Reply:
2a2e4e6 makes `ccframe` print a message (and exit) if pandas can't be imported.

Reply:
Right now, all of the bridges fail silently in this way if the external dependency isn't present.